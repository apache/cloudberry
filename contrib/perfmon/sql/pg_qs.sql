-- start_ignore
\! sudo /bin/bash -c 'source /usr/local/cloudberry-db-devel/greenplum_path.sh;pip3 install psycopg2;pip3 install progressbar'
-- end_ignore
\!python3 src/gpmon/tests/pg_qs_test_runner.py --port $PGPORT --database gpperfmon --user gpadmin
