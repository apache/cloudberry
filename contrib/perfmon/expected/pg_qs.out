\!python3 src/gpmon/tests/pg_qs_test_runner.py --port $PGPORT --database gpperfmon --user gpadmin
setting up...
done!
test when two backends try to extract state of each other...
ok!
test statistics of simple query...
ok!
test when two backends compete with each other to extract state from third running backend...
ok!
test statistics under calling function...
ok!
test plan costs...
ok!
test timing statistics...
ok!
test statistics on conflicting tuples under INSERT ON CONFLICT query...
ok!
test skip explain analyze query...
ok!
test statistics of init plan not supported now...
ok!
test qe can cache query state when finished...
ok!
tearing down...
done!
